### DISCUSSION TITLE
Failed to login and turn on privacy mode when "Auto Login" and "Privacy Mode(2)" are both on

### DISCUSSION THREAD

**Discussion by xiongsp** [rustdesk/rustdesk#9350] [General]
### Bug Description

It's likely unable to login and turn on privacy mode when "Auto Login" and "Privacy Mode(2)" are both on. Mode 1 is not effected.

### How to Reproduce

1. Turn on "Auto Login" and "Privacy Mode(2)"
2. Try to connect to PC

### Expected Behavior

"Auto Login" and "Privacy Mode(2)" are successfully executed.

### Operating system(s) on local side and remote side

Win11 -> Win10

### RustDesk Version(s) on local side and remote side

1.3.0 -> 1.3.0

### Screenshots

In fact, it sometimes pop up a rustdesk error window and then disappear too quickly to catch it. And it's really hard to reproduce.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/7105#issuecomment-1938005828

**Reply by xiongsp** [reply]
Thanks for replying soon. I've read this before I send the issue. In fact auto login often works correctly but privacy mode doesn't.

**Reply by fufesou** [reply]
Please try the latest nightly build.  https://github.com/rustdesk/rustdesk/pull/9141

> But when we plug in a virtual display again, the controlled end will only display the virtual display and turn off the physical display.
Because the display combination setting is still to turn off the physical display.
> The user must manually change the display settings.

Please also plug in a virtual display, and restore the display settings mannually. If you find the settings is not "Extend desktop to this display".

<img width="253" alt="1726270835686" src="https://github.com/user-attachments/assets/e62d556b-3fe2-42cd-b5de-027c2b9f31b4">

![image](https://github.com/user-attachments/assets/e25f53ed-a10e-4976-b6f4-068166f2ad4d)
